model_list_dir/SIFC.json

```diff
@@ -1,14 +1,17 @@
 {
-  "authors": "Lisa Colburn",
+  "authors": "Lisa Colburn and Mike Jepson",
   "background_text": "NOAA Fisheries Community Social Vulnerability Indicators Toolbox is currently comprised of a suite of 14 statistically robust social, economic, and climate change indicators that uniquely characterize and evaluate a community’s vulnerability and resilience to disturbances (regulations, extreme weather, oil spills, sea level rise, etc.). The indicator map and graphing tool enables users to analyze both environmental justice as well as the climate vulnerability of over 4,600 coastal communities in 24 states. The social indicators are also routinely used for National Environmental Policy Act (NEPA), Magnuson-Stevens Act (MSA) and Executive Order 12898 (environmental justice) analyses.",
-  "maintainer_name": "Fisheries Integrated Toolbox",
-  "maintainer_email": "fisheries.toolbox@noaa.gov",
+  "maintainer_name": "Lisa Colburn",
+  "maintainer_email": "lisa.colburn@noaa.gov",
   "tool_abbreviation": "SIFC",
   "tool_name": "Social Indicators for Fishing Communities",
   "online_app_link": "https://www.st.nmfs.noaa.gov/data-and-tools/social-indicators/",
+  "website_link": "https://www.fisheries.noaa.gov/national/socioeconomics/social-indicators-fishing-communities-0",
+  "documentation_link": "https://www.fisheries.noaa.gov/national/socioeconomics/social-indicator-supporting-information",
+  "citation": "NOAA Fisheries Office of Science and Technology. 2020. NOAA Fisheries Community Social Vulnerability Indicators (CSVIs). Version 4 (Last updated: September 20, 2023).", 
   "active_development": true,
   "noaa_internal": true,
   "uses_github_releases": false,
   "toolbox_drawers": ["Human Dimensions"], 
-  "keywords": ["Web App", "Data Exploration", "Socioeconomics"]
-}
\ No newline at end of file
+  "keywords": ["Web App", "Data Exploration", "Socioeconomics", "Indicators", "Fishing Engagement and Reliance"]
```

Comment:
@lpfeif these are the current keywords being used on the toolbox: https://github.com/noaa-fisheries-integrated-toolbox/toolbox_web_templating/blob/main/json_analysis/keywords.txt

Do any of the existing one work? If you feel like a new one is needed, we can add it though.

Also, sorry there was no documentation on the keywords - I'll add some text to the readme!